.github/workflows/sdk-publish.yaml

```diff
@@ -258,6 +258,35 @@ jobs:
           footer: "Linked Repo <{repo_url}|{repo}> | <{run_url}|View Run>"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  publish-nuget:
```

Comment:
I think you're missing a few things from this file--if you just pattern match to `publish_java` you'll see what you need to add. Also for consistency this should just be `publish-csharp` instead of `publish-nuget`

Reply:
Good catch; I did that for the generation yaml, but forgot for publish.

> Also for consistency this should just be publish-csharp instead of publish-nuget

That's what I had initially, but all of the others are `publish-npm` or `publish-pypi` (terraform is also `publish-terraform`, so 🤷‍♀️ ).

Reply:
Also i think you need to add `nuget_api_key` to the list of inputs at the top

Reply:
yup, just did that too

Reply:
Oh ya that's fine then 👍